earth_enterprise/src/SConstruct

```diff
@@ -150,16 +150,27 @@ else:
   python_include = '/usr/include/python%s' % python_major_version
 
 # setup platform specific operations
-is_hardy = True
+is_min_ubuntu = True
 # if non-zero return status it is not hardy
```

Comment:
This comment block is out of sync with the code, now. Presumably it should just be removed?

Reply:
I dont follow

Reply:
Sorry Jim, I misunderstood your comment. No issues, will change.